wolf-daemon/wolf_daemon.py

```
@@ -0,0 +1,249 @@
+        url = f"{self.api_base}/sendMessage"
+        payload = {
+            "chat_id": self.chat_id,
+            "text": text,
+            "parse_mode": parse_mode
+        }
+
+        try:
+            async with aiohttp.ClientSession() as session:
+                async with session.post(url, json=payload) as resp:
+                    if resp.status == 200:
+                        logger.info(f"✅ Message sent to Telegram")
```

Ruff (0.14.1)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)